ecs/x.go

```diff
@@ -17,17 +17,18 @@
 package ecs
 
 const (
-	extensionSecurityGroup   = "x-aws-securitygroup"
-	extensionVPC             = "x-aws-vpc"
-	extensionPullCredentials = "x-aws-pull_credentials"
-	extensionLoadBalancer    = "x-aws-loadbalancer"
-	extensionProtocol        = "x-aws-protocol"
-	extensionCluster         = "x-aws-cluster"
-	extensionKeys            = "x-aws-keys"
-	extensionMinPercent      = "x-aws-min_percent"
-	extensionMaxPercent      = "x-aws-max_percent"
-	extensionRetention       = "x-aws-logs_retention"
-	extensionRole            = "x-aws-role"
-	extensionManagedPolicies = "x-aws-policies"
-	extensionAutoScaling     = "x-aws-autoscaling"
+	extensionSecurityGroup         = "x-aws-securitygroup"
+	extensionVPC                   = "x-aws-vpc"
+	extensionPullCredentials       = "x-aws-pull_credentials"
+	extensionLoadBalancer          = "x-aws-loadbalancer"
+	extensionCrossZoneLoadBalancer = "x-aws-cross_zone_loadbalancer"
```

Comment:
I'd prefer we avoid custom extensions which are 100% AWS-specific and won't ever be candidates for compose-spec. We already have too much of those.

We need to clarify if this option is a "sane default" we should enable by default.

Reply:
@mreferre can you please confirm this option should just be set by default ?
I'd prefer we set "sane defaults" and introduce more flexibility later if users ask for it, vs preemptively adding too much configuration options which make newcomer experience harder.

Reply:
@ndeloof Sorry I forgot to update you on this. I have contacted the NLB team and they confirmed that the default behavior is there for historical reasons and we should enable cross-AZ traffic.